StripeUICore/StripeUICore/Source/Elements/PickerField/PickerFieldView.swift

```diff
@@ -194,7 +194,7 @@ final class PickerFieldView: UIView {
     }
 
     override func becomeFirstResponder() -> Bool {
-        guard _canBecomeFirstResponder else {
+        guard _canBecomeFirstResponder, isUserInteractionEnabled else {
```

Comment:
It's to fix a weird visual bug that was occurring. It was only occurring in CustomerSheet, but for some reason when we could toggle `isUserInteractionEnabled` the OS would for some reason, invoke `becomeFirstResponder` on the DropDownFieldElement, causing it to display itself. I think this is a bug of sorts that I have ran into before but can't find any other information online regarding it. So this is an additional safe guard so that doesn't happen.
 
> re: break this out into a separate commit

Yeah might be a little more useful for historical reasons, but I feel that asserting that `isUserInteractionEnabled` is true before becoming the first responder is non-controversial.

I added a comment for clarity.